install/roles/qpc_server/tasks/main.yml

```diff
@@ -31,41 +31,89 @@
   when:
     - open_port
 
-- name: Determine if supervisord should be used
+- name: Determine if supervisord should be used inside of Docker container
   set_fact:
     use_supervisord: "{{ use_supervisord | bool | default(true) }}"
   when: use_supervisord is defined
 
-- name: Defaulting server install setting
+- name: Defaulting server supervisord setting to true
   set_fact:
     use_supervisord: true
   when: use_supervisord is not defined
 
-- name: Set default server url
+- name: Set the server version to install using user argument
   set_fact:
-    default_server_url: "https://github.com/quipucords/quipucords/releases/download/{{RELEASE_TAG}}/quipucords.{{RELEASE_TAG}}.tar.gz"
+    install_flag_server_version: "{{ server_version }}"
+  when:
+    - server_version is defined
+
+- name: Set the server docker image tag to user specified version
+  set_fact:
+    server_image_tag: "{{ install_flag_server_version }}"
+  when:
+    - install_flag_server_version is defined
+
+- name: Default the server docker tag to unknown
+  set_fact:
+    server_image_tag: "unknown"
+  when:
+    - install_flag_server_version is not defined
+
+- name: Query GitHub for the latest release information
+  uri:
+    url: https://api.github.com/repos/quipucords/quipucords/releases/latest
+    validate_certs: no
+    return_content: yes
+    headers:
+      Content-Type: "application/json"
+    body_format: json
+  retries: 5
+  delay: 5
+  register: github_release_info
+  until: github_release_info.status == 200 or github_release_info.status == 401
+  changed_when: false
+  when:
+    - install_flag_server_version is not defined
+
+- name: Parse release tag from github JSON
+  set_fact: 
+    github_release: "{{ github_release_info.json.tag_name }}"
+  when: 
+    - install_flag_server_version is not defined
+    - github_release_info is defined
+    - github_release_info.json is defined
+    - github_release_info.json.tag_name is defined
+
+- name: Override default server_image_tag value with release value
+  set_fact:
+    server_image_tag: "{{ github_release }}"
+  when:
+    - install_flag_server_version is not defined
+    - github_release is defined
+
+- name: Set the quipucords release server image file name
+  set_fact:
+    github_server_filename: "quipucords_server_image.tar.gz"
 
-- name: Set server url
+- name: Set the quipucords release server image file name
   set_fact:
-    server_url: "{{ ansible_env['SERVER_URL'] |  default(default_server_url)}}"
-  when: server_url is not defined
+    release_cache_server_filename: "quipucords.{{ server_image_tag }}.tar.gz"
 
-- name: Set server image name
+- name: Set server docker image name
   set_fact:
-    image: "{{ ansible_env['SERVER_IMAGE'] | default('quipucords') }}"
-  when: image is not defined
+    server_docker_image_name: "quipucords"
 
-- name: Set server_image_and_tag
+- name: Set Quipucords docker image and tag
```

Comment:
not a blocker at all but there are a lot of inconsistencies within the name descriptions, for example, Quipucords vs quipucords, Postgres vs postgres

Reply:
Fixed